docs/content/latest/cdc/cdc_documentation_v1.md

```diff
@@ -0,0 +1,188 @@
+| update_min_cdc_indices_interval_secs              | 60                | This specifies how often to read the cdc_state table to get the minimum applied index for each tablet across all streams. This information is used to correctly keep log files that contain unapplied entries. This is also the rate at which a tablet's minimum replicated index across all streams is sent to the other peers in the configuration. If flag enable_log_retention_by_op_idx is disabled, this flag has no effect. |
+| cdc_ybclient_reactor_threads                      | 50                | The number of reactor threads to be used for processing ybclient requests for CDC. Increase to improve throughput on large tablet setups.                                                                                                                                                                                                                                                                                          |
+| enable_log_retention_by_op_idx                    | true              | If true, logs will be retained based on an op id passed by the cdc service.                                                                                                                                                                                                                                                                                                                                                        |
+| log_max_seconds_to_retain                         | 86400             | Log files that are older will be deleted even if they contain cdc unreplicated entries. If 0, this flag will be ignored. This flag is ignored if a log segment contains entries that haven't been flushed to RocksDB.                                                                                                                                                                                                              |
+| log_stop_retaining_min_disk_mb                    | 102400            | Stop retaining logs if the space available for the logs falls below this limit. This flag is ignored if a log segment contains unflushed entries.                                                                                                                                                                                                                                                                                  |
+  
+### Running the Debezium connector
+  
+  Head over to the [Debezium connector](https://github.com/vaibhav-yb/cdc-docs/blob/main/running_cdc_with_debezium.md) doc for the steps on how to run with Debezium connector.
```

Comment:
@vaibhav-yb @polarweasel - link to Debezium steps would need to be updated.

Reply:
This can be a relative link once that page is in the right spot.